FashionData/maestro
# [Feat] - User profile & theming configuration

## Related issue(s)

Closes #7 | Closes #8 | Closes #22 

## Description

Integrate profile view & add a way to provide custom theme from consumer application.


## What does this PR do?

- [x] Add shared styles into all the application
- [x] Add user information into firebase project
- [x] Add `connections-history` into firebase project
- [x] Add the possibility to provide a custom `element-ui` theme (conserving the possibility to use `scss` variables)
- [x] Add doc concerning theming


## Deploy notes

Review & merge.


## Steps to test or reproduce

1. Go to `/profile` => You should see the new UI of the page
2. Into the `dev` environment, add a `theme` option while calling `initializeApp` => Global theme should be override


## Impacted area(s) in Application

- Style
- Profile
